doc/languages-frameworks/python.section.md

```diff
@@ -2017,10 +2017,12 @@ The following rules are desired to be respected:
 * `meta.platforms` takes the default value in many cases.
   It does not need to be set explicitly unless the package requires a specific platform.
 * The file is formatted with `nixfmt-rfc-style`.
-* Commit names of Python libraries should reflect that they are Python
-  libraries, so write for example `python311Packages.numpy: 1.11 -> 1.12`.
-  It is highly recommended to specify the current default version to enable
-  automatic build by ofborg.
+* Commit names of Python libraries must reflect that they are Python
+  libraries (e.g. `python311Packages.numpy: 1.11 -> 1.12`).
+* The current default version of python should be included
```

Comment:
Since both are commit message conventions, I want to put them in one item.

Reply:
I'd argue that while they deal with the same subject matter (commit messages), they are separate conventions and belong in separate entries.

Reply:
Okay. So how about this?
```diff
 * Commit names of Python libraries must reflect that they are Python
-   libraries (e.g. `python311Packages.numpy: 1.11 -> 1.12`).
+   libraries (e.g. `python311Packages.numpy: 1.11 -> 1.12` rather than `numpy: 1.11 -> 1.12`).
 * The current default version of python should be included
```
I think it would make it clearer how the first item differs from the second.

Reply:
> … must reflect their package set relationship

Reply:
Good suggestion. Done!